cli/src/package.rs

```diff
@@ -80,8 +111,10 @@ impl PackageCommand {
             Command::DebugResolution => {
                 let path = self.find_manifest()?;
                 let manifest = ManifestFile::from_path(path.clone())?;
-                let snap = manifest.snapshot_dependencies(config)?;
-                let package_map = snap.package_map(&manifest)?;
+                let snap = manifest.snapshot_dependencies(config.clone())?;
+                let index = PackageIndex::shared(config.clone())?;
+                let resolution = resolve::resolve(&manifest, snap, index, config)?;
+                let package_map = resolution.package_map(&manifest)?;
                 eprintln!("{package_map}");
```

Comment:
left-over debug statement?

Reply:
No, printing the package map is the main job of this command. (It maybe be time to remove the command, though. It was mainly just to help debug things while implementing...)